testcases/kernel/containers/netns/netns_helper.h

```diff
@@ -64,7 +69,7 @@ static void check_netns(void)
 		tst_brkm(TCONF | TERRNO, NULL, "CLONE_NEWNS (%d) not supported",
 			 CLONE_NEWNS);
 
-	pid = do_clone_unshare_test(T_UNSHARE, CLONE_NEWNET | CLONE_NEWNS, NULL,
+	pid = do_clone_unshare_test(T_UNSHARE, CLONE_NEWNET | CLONE_NEWNS, dummy,
 			NULL);
```

Comment:
As far as I can tell it passes wrong number of arguments as well, we should pass two more ```NULL```s there as well.

Also can you describe where it segfaults, i.e. architecture, distribution, etc. in the commit message?